### DISCUSSION TITLE
Route Sharing Feedback

### DISCUSSION THREAD

**Discussion by sandermvanvliet** [sandermvanvliet/RoadCaptain#116] [General]
This is the place for feedback on the new Route Sharing feature in RoadCaptain.

If you've found a bug please create an [issue](https://github.com/sandermvanvliet/RoadCaptain/issues/new?assignees=&labels=routebuilder%2C+runner%2C+route-sharing%2C+triage&projects=&template=route-sharing-bug.md&title=) and mention it here 👍 

If you've got a cool idea or questions you are definitely in the right place 👍

**Comment by RoobyDo** [comment]
Is there any way to use a route I created for a Custom Meetup on Zwift? A Custom Meetup allows me to use the rubber band feature so riders of different abilities all stay together regardless of rider watts/output. If I shared the route with 1 or 2 people and we all started at same time (using Road Runner), we could stay together. But, when I get 20+ riders, it's almost impossible to keep the group together - especially on long routes; like doing all-of-Watopia in one ride. Thoughts?

**Comment by sandermvanvliet** [comment]
I haven’t gotten around to testing RoadCaptain for meet ups yet. But I definitely want to… at some point. 

It’s a question of when I’ve got time available to look into it

**Reply by MarcoEkelo1973** [reply]
Found this application recently, added a lot of rebel routes to the repository. Works like a charm ;-)

One question: when will Scotland be available ? 

Keep it the good work
